stack/stack.go

```diff
@@ -1,39 +1,91 @@
 package main
 
+//Imported Libraries
 import (
-	"errors"
 	"fmt"
 )
 
-// Stack implements stack DS
-type Stack struct {
-	list []int
+//Constants
+const x  = 10
```

Comment:
Formatting can be improved.

*Origin: GofmtBear, Section: `gofmt`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgx2e53q9/stack/stack.go
+++ b/tmp/tmpgx2e53q9/stack/stack.go
@@ -6,7 +6,7 @@
 )
 
 //Constants
-const x  = 10
+const x = 10
 
 //Implementation of the stack structure
 type stack struct {
```